### PR TITLE
CubeCamera, WebGLRenderTarget, WebGLRenderTargetCube Fix.

### PR DESCRIPTION
**Pull request by prafullit**
clone support for CubeCamera, WebGLRenderTargetCube.

### CODE REVIEW

File: src/cameras/CubeCamera.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
  *	- renders scene into axis-aligned cube
  *
  * @author alteredq / http://alteredqualia.com/
+ * @author prafullit
```
Comment (WestLangley) [inline src/cameras/CubeCamera.js]: Add your name as an author only if you feel your contribution is significant, and you are willing to be responsible for the file maintenance.

File: src/cameras/CubeCamera.js [hunk 2]
```diff
@@ -82,3 +83,14 @@ THREE.CubeCamera = function ( near, far, cubeResolution ) {
 
 THREE.CubeCamera.prototype = Object.create( THREE.Object3D.prototype );
 THREE.CubeCamera.prototype.constructor = THREE.CubeCamera;
+
+THREE.CubeCamera.prototype.clone = function () {
+
+	var camera = new THREE.CubeCamera();
+
+	THREE.Object3D.prototype.clone.call( this, camera, false );
+	this.renderTarget.clone(camera.renderTarget);
```
Comment (WestLangley) [inline src/cameras/CubeCamera.js]: I am not sure a clone method is needed here. But even if @mrdoob wants it, I do not believe this code will clone the children properly.

File: src/renderers/WebGLRenderTarget.js [hunk 3]
```diff
@@ -49,9 +49,12 @@ THREE.WebGLRenderTarget.prototype = {
 		}
 	},
 
-	clone: function () {
+	clone: function (tmp) {
```
Comment (WestLangley) [inline src/renderers/WebGLRenderTarget.js]: We need to discuss if `clone()` shoud take an argument. I do not think it should -- it becomes too much like `copy()`. /ping @mrdoob

File: src/renderers/WebGLRenderTargetCube.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 /**
  * @author alteredq / http://alteredqualia.com
+ * @author prafullit
```
Comment (WestLangley) [inline src/renderers/WebGLRenderTargetCube.js]: ditto

**Comment by WestLangley** [comment]
@prafullit Thank you for noticing these issues. We have not been consistent in our use of the `clone()` method. We just need to come to an agreement on the behavior we want.

**Comment by prafullit** [comment]
Yeah, I too have noticed that. 
And also if a user is able to access clone, I think it should work as expected.
Maybe a consistent approach should be thought of.

**Pull request closed without merging**